src/chrome/content/rules/Kayak.xml

```diff
@@ -1,10 +1,77 @@
 <ruleset name="Kayak">
-  <target host="kayak.com" />
-  <target host="www.kayak.com" />
-  <target host="www.kayak.co.uk" />
-  <target host="kayak.co.uk" />
+	<target host="kayak.com" />
+	<target host="www.kayak.com" />
 
-  <rule from="^http://(?:www\.)?kayak\.com/" to="https://www.kayak.com/"/>
-  <rule from="^http://(?:www\.)?kayak\.co\.uk/" to="https://www.kayak.co.uk/"/>
-</ruleset>
+	<target host="kayak.ae" />
+	<target host="www.kayak.ae" />
+	<target host="kayak.ch" />
+	<target host="www.kayak.ch" />
+	<target host="kayak.de" />
+	<target host="www.kayak.de" />
+	<target host="kayak.dk" />
+	<target host="www.kayak.dk" />
+	<target host="kayak.es" />
+	<target host="www.kayak.es" />
+	<target host="kayak.fr" />
+	<target host="www.kayak.fr" />
+	<target host="kayak.ie" />
+	<target host="www.kayak.ie" />
+	<target host="kayak.it" />
+	<target host="www.kayak.it" />
+	<target host="kayak.nl" />
+	<target host="www.kayak.nl" />
+	<target host="kayak.no" />
+	<target host="www.kayak.no" />
+	<target host="kayak.pl" />
+	<target host="www.kayak.pl" />
+	<target host="kayak.pt" />
+	<target host="www.kayak.pt" />
+	<target host="kayak.ru" />
+	<target host="www.kayak.ru" />
+	<target host="kayak.se" />
+	<target host="www.kayak.se" />
+	<target host="kayak.sg" />
+	<target host="www.kayak.sg" />
+
+	<target host="kayak.com.ar" />
+	<target host="www.kayak.com.ar" />
+	<target host="kayak.com.au" />
+	<target host="www.kayak.com.au" />
+	<target host="kayak.com.br" />
+	<target host="www.kayak.com.br" />
+	<target host="kayak.com.hk" />
+	<target host="www.kayak.com.hk" />
+	<target host="kayak.com.mx" />
+	<target host="www.kayak.com.mx" />
+	<target host="kayak.com.my" />
+	<target host="www.kayak.com.my" />
+	<target host="kayak.com.tr" />
+	<target host="www.kayak.com.tr" />
 
+	<target host="ca.kayak.com" />
+	<target host="www.ca.kayak.com" />
+	<target host="cn.kayak.com" />
+	<target host="www.cn.kayak.com" />
+	<target host="fi.kayak.com" />
+	<target host="www.fi.kayak.com" />
+	<target host="gr.kayak.com" />
```

Comment:
@loveithateit I'm not sure what you mean by "rewrite to", but would adding
`<rule from="^http://gr.kayak.com" to="https://www.gr.kayak.com" />`
solve it?